stable/sonarqube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 2.2.0
+version: 2.2.1
```

Comment:
Please bump the minor version instead as this adds a feature

Reply:
Version updated to 2.3.0.